infra/modules/aws-cognito-identity-cli/variables.tf

```diff
@@ -13,6 +13,9 @@ variable "aws_role" {
   description = "Role to assume during script execution"
 }
 
-variable "login-ids" {
+# TODO: expect value format "${module.cognito-identity-pool.developer_provider_name}=${module.msft-connection[k].connector.application_id}"
+# --> add validation of that OR split those components in properties of an object??
+variable "login_ids" {
```

Comment:
@aperez-worklytics changed this to `_`, as I think better style to support (potential) invocation of terraform configurations via command line with variables, eg https://developer.hashicorp.com/terraform/language/values/variables#variables-on-the-command-line